middleware/authenticateJWT.js

```diff
@@ -0,0 +1,27 @@
+const accessTokenSecret = 'RBK_TOKEN';
+const jwt = require('jsonwebtoken');
+module.exports = {
+    async authenticateJWT(req, res, next) {
+        const authHeader = req.headers.authorization;
+
+        if (authHeader) {
+            const token = authHeader.split(' ')[1];
+
+            jwt.verify(token, accessTokenSecret, (err, user) => {
+                if (err) {
+                    return res.send({
+                        message: 'you are not autherized '
+                    });
+                }
+
+                req.user = user;
+                next();
+            });
+        } else {
+            res.send({
+                message: 'you are not logged in '
+            });
+        }
+    }
+
+}
```

Comment:
This file should be a service not a Middleware
The service is than consumed by an authorizatin operation that will be called by the controller that will be put inside every route